internal/pkg/policy/policy_output_test.go

```diff
@@ -0,0 +1,175 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package policy
+
+import (
+	"context"
+	"testing"
+
+	"github.com/elastic/fleet-server/v7/internal/pkg/bulk"
+	"github.com/elastic/fleet-server/v7/internal/pkg/model"
+	"github.com/elastic/fleet-server/v7/internal/pkg/smap"
+	ftesting "github.com/elastic/fleet-server/v7/internal/pkg/testing"
+	"github.com/rs/zerolog"
+)
+
+var TestPayload []byte
+
+func TestPolicyLogstashOutputPrepare(t *testing.T) {
+	bulker := ftesting.NewMockBulk(&bulk.ApiKey{
+		Id:  "test id",
+		Key: "test key",
+	})
+	po := PolicyOutput{
+		Type: OutputTypeLogstash,
+		Name: "test output",
+		Role: &RoleT{
+			Sha2: "fake sha",
+			Raw:  TestPayload,
+		},
+	}
+
+	err := po.Prepare(context.Background(), zerolog.Logger{}, bulker, &model.Agent{}, smap.Map{}, false)
+	if err != nil {
```

Comment:
I think we have the required module in fleet-server to do `require.NoNit(t, err, "expect....")`, also what is expected here, we check we aren't returning an error but we do not check for any transformation result?